presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/PhaseSpecification.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.framework;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+
+public class PhaseSpecification
+{
+    public enum ExecutionStrategy
+    {
+        STREAM,
+        CONCURRENT
+    }
+
+    private final ExecutionStrategy executionStrategy;
+    private final List<List<String>> streams;
+
+    @JsonCreator
+    public PhaseSpecification(
+            @JsonProperty("executionStrategy") ExecutionStrategy executionStrategy,
+            @JsonProperty("streamConfiguration") List<List<String>> streams)
+    {
+        this.executionStrategy = executionStrategy;
+        this.streams = ImmutableList.copyOf(streams);
```

Comment:
Do a deep copy because it is a nested list.
```
streams.stream()
   .map(ImmutableList::copyOf)
   .collect(toImmutableMap());
```

Reply:
I think you meant  streams.stream().map(ImmutableList::copyOf).collect(toList());

Reply:
Oh, I meant `toImmutableMap `